apps/uci/web/src/components/common/starred-chat-list/index.tsx

```diff
@@ -52,134 +95,103 @@ export const StarredChatList: FC<{ user: User }> = ({ user }) => {
                                 src={vidUrl}
                             />
-                            <div
-                                style={{
-                                    display: 'flex',
-                                    justifyContent: 'space-between',
-                                    alignItems: 'self-end'
-                                }}
-                            >
-                                <span style={{ color: 'var(--grey)', fontSize: '10px' }}>
+                            <div>
+                                <span>
                                     {moment
                                         .utc(content?.data?.sentTimestamp || content?.data?.receivedTimeStamp)
                                         .local()
                                         .format('DD/MM/YYYY : hh:mm')}
                                 </span>
                             </div>
                         </div>
-                    </Bubble>
+                    </StyledBubbleImage>
                 );
             }
 
             case 'file':
                 return (
                     <FileCard
-                        // @ts-ignore
                         file={{
                             name: 'sample',
-                            size: 12345
+                            size: 12345,
                         }}
-                        extension="pdf"
-                    >
-                        <a
-                            target="_blank"
-                            href="https://www.africau.edu/images/default/sample.pdf"
-                            rel="noreferrer"
-                        >
-                            Sample
+                        extension="pdf" >
+                        <a target="_blank"
+                            href={content?.data?.payload?.media?.url || content?.data?.fileUrl}
+                            style={{ textDecoration: 'none' }}>
```

Comment:
also ,previously download was handled from android ,so can you handle it from web?